anomaly_detection/ml/algorithms/gaussian.py

```diff
@@ -57,30 +63,32 @@ def __init__(self):
         super(Gaussian, self).__init__(algorithm_name=contants.GAUSSIAN_MODEL)
 
     def create_training(self, training):
-        cv_data = ds.read('../dataset/performance-cv.csv')
-        gt_data = ds.read('../dataset/performance-gt.csv')
-        tr_data = ds.read('../dataset/performance-tr.csv')
+        data_path = CONF.apiserver.train_dataset_path
+        cv_data = ds.read(os.path.join(data_path, 'performance-cv.csv'))
+        gt_data = ds.read(os.path.join(data_path, 'performance-gt.csv'))
+        tr_data = ds.read(os.path.join(data_path, 'performance-tr.csv'))
```

Comment:
Please add description to "cv", "gt", "tr".  I assume "tr" means "training".  What is "cv" and "gt"?

Reply:
Also add TODO that this code will be modified so that the data can be read from database.